public/app/core/components/Select/FolderPicker.tsx

```diff
@@ -16,6 +16,7 @@ import { AccessControlAction, PermissionLevelString } from 'app/types';
 export type FolderPickerFilter = (hits: DashboardSearchHit[]) => DashboardSearchHit[];
 
 export const ADD_NEW_FOLER_OPTION = '+ Add new';
+export const SLICE_RESULTS_TO = 1000;
```

Comment:
I'm thinking it might be useful to link the original issue or [PR description ](https://github.com/grafana/grafana/pull/61221#issue-1527230780) here so that anyone that wonders about this slicing can have some context given this is a shared component.